### PR TITLE
Add POD for advise types and propse change to the notifications

### PR DESCRIPTION
**Pull request by bdraco**
in 56.0.15+

### CODE REVIEW

File: pkg/Cpanel/Security/Advisor.pm [hunk 1]
```diff
@@ -26,6 +26,36 @@ package Cpanel::Security::Advisor;
...
+    );
+
+    my $msgs = $comet->get_messages($CHANNEL);
+    foreach my $msg ( @{$msgs} ) {
+        my $msg_ref = Cpanel::AdminBin::Serializer::Load($msg);
+
+        ....
+
+    }
+=cut
```
Comment (ColMelvin) [inline pkg/Cpanel/Security/Advisor.pm]: There is no empty line before `=cut`.  While this really shouldn't be a problem, some POD parsers fail under this condition.

File: pkg/Cpanel/Security/Advisor.pm [hunk 2]
```diff
@@ -40,6 +70,54 @@ our $ADVISE_INFO = 2;
 our $ADVISE_WARN = 4;
 our $ADVISE_BAD  = 8;
 
+=head1 ADVISE TYPES
+
+=head2 ADVISE_GOOD
+
+=over
+
+Changes DO NOT send iContact notices)
```
Comment (ColMelvin) [inline pkg/Cpanel/Security/Advisor.pm]: `s/)//`

File: pkg/Cpanel/Security/Advisor.pm [hunk 3]
```diff
@@ -40,6 +70,54 @@ our $ADVISE_INFO = 2;
...
+
+=back
+
+All is well
+
+=head2 ADVISE_INFO
+
+=over
+
+Changes send iContact notices for 11.56.0.14 and below, Changes DO NOT send iContact notices for 11.56.0.15 and above)
```
Comment (ColMelvin) [inline pkg/Cpanel/Security/Advisor.pm]: `s/)//`

**Comment by bdraco** [comment]
@ColMelvin   I repushed the branch to address the issues

**Pull request merged**